src/annotator/test/annotation-counts-test.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var annotationCounts = require('../annotation-counts');
+
+describe('annotationCounts', function () {
+  var countEl1;
+  var countEl2;
+  var CrossFrame = null;
+  var fakeCrossFrame = {};
+  var sandbox = sinon.sandbox.create();
+
+  beforeEach(function () {
+    countEl1 = document.createElement('button');
+    countEl1.setAttribute('data-hypothesis-annotation-count');
+    document.body.appendChild(countEl1);
+
+    countEl2 = document.createElement('button');
+    countEl2.setAttribute('data-hypothesis-annotation-count');
+    document.body.appendChild(countEl2);
+
+    fakeCrossFrame.on = sandbox.stub().returns(fakeCrossFrame);
+    
+    CrossFrame = sandbox.stub();
+    CrossFrame.returns(fakeCrossFrame);
+  });
+
+  afterEach(function () {
+    sandbox.restore();
+    countEl1.parentElement.removeChild(countEl1);
+    countEl2.parentElement.removeChild(countEl2);
+  });
+
+  describe('listen for "publicAnnotationCountChanged" event', function () {
+    var emitEvent = function () {
```

Comment:
I presume `emitEvent` is what the CoffeeScript two-liner turned into when compiled to ES5? I see why you wanted to put it in a shared module ;). It's fine for the moment. We can simplify it somewhat when ES6 is supported in the client.